README.md

```diff
@@ -66,31 +66,74 @@ JSONObject result = CoreApi.chargeTransaction(param);
 // SnapApi request with global config
 JSONObject result = SnapApi.createTransaction(param);
 ```
-
-All the request can accept an optional Config object. This is used if you want to set an others' config like idempotency key, proxy,
-or if you want to pass the server-key on each method, or use multiple account on each method.
+#### Per-request Configuration
+It is also possible if on each of individual requests you want to set a unique/different configuration; like idempotency key, 
+proxy, override/append notification url, or multiple account API keys via Config options and set as param on static method e.g: `CoreApi.chargeTransaction(param, coreApiConfigOptions)` method available on `CoreApi.class`, `SnapApi.class`, `TransactionApi.class` or `IrisApi.class`. Please follow the steps given below.
```

Comment:
I think you missed/misunderstood [this comment](https://github.com/Midtrans/midtrans-java/pull/18#discussion_r887538034). Can you add sentence here that say:

> Note: This method is expected to be thread-safe, you should prefer this if you are implementing multi-threading/concurrency in your code.